telegram/ext/_chatjoinrequesthandler.py

```diff
@@ -43,6 +44,14 @@ async def callback(update: Update, context: CallbackContext)
 
             The return value of the callback is usually ignored except for the special case of
             :class:`telegram.ext.ConversationHandler`.
+        chat_id (:obj:`int` | Collection[:obj:`int`], optional): Filters requests to allow only
+            those which are from specified chat ID(s).
```

Comment:
```suggestion
            those which are asking to join the specified chat ID(s).
```

or similar wording. My intention here is to make it more clear that `chat_join_request.chat.id` is filtered, not `chat_join_request.from_user.id` :)

Same for `username`

Reply:
nice! Important distinction imo